src/More/More.js

```diff
@@ -0,0 +1,142 @@
+import React, { useState, Fragment } from 'react';
```

Comment:
I will change the architecture / organisation of the component.
I will organize it like:
- More
  |--MoreAvatar
  | .    |--MoreAvatar.js
  | .    |--index.js
  |--MoreContent
         |--MoreContent.js
         |--index.js